README.md

```diff
@@ -20,22 +20,23 @@
   <a href="https://slack.meilisearch.com"><img src="https://img.shields.io/badge/slack-MeiliSearch-blue.svg?logo=slack" alt="Slack"></a>
 </p>
 
-<p align="center">⚡ Lightning Fast, Ultra Relevant, and Typo-Tolerant Search Engine MeiliSearch client written in Go</p>
+<p align="center">⚡ The MeiliSearch API client written for Golang</p>
```

Comment:
I also changed this sentence that I found unreadable. WDYT?

Reply:
I agree, and just below we explain what MeiliSearch is, just in case!

Reply:
I agree, and just below we explain what MeiliSearch is, just in case!

Reply:
@bidoubiwa what do you think about my change here? Is it weird to you like the description below?